index.html

```diff
@@ -64,8 +69,16 @@ <h2><a href="{{conf.link}}">{{conf.name}} {{conf.year}}</a></h2>
               <div class="col-xs-12 col-sm-6">
                 <span class="timer"></span>
                 <div class="deadline">
-                  <div>Deadline:
-                    <span class="deadline-time"></span>
+                  <div>
+                      {% if num_deadlines >=2 %}
+                      {% if conf.comment %}<br />{% endif %}
```

Comment:
Any reason why you prefer to break if a comment exists?
I deployed locally without this and I think it looks better

Reply:
You are right. I think this was because I originally put the `Deadline (M / N)` in front of the comment and this is probably a leftover. I removed it.